docs/source/common/trainer.rst

```diff
@@ -1168,15 +1170,58 @@ will still show torch.float32.
     trainer = Trainer(precision=32)
 
     # 16-bit precision
-    trainer = Trainer(precision=16, gpus=1)
+    trainer = Trainer(precision=16)
```

Comment:
below it says 16-bit is not supported on cpu, so I guess it would be best to keep the gpus argument here :)